### DISCUSSION TITLE
Mysql support

### DISCUSSION THREAD

**Discussion by lvjoemcintyre** [paillave/Etl.Net#357] [Ideas]
Are there plans for mysql support? 

I started trying to modifiy the sqlserver project to support mysql. 
I'm bumping into some trouble understanding how to debug it.

**Comment by paillave** [comment]
Hello,
It makes part of support that are planned for much later. If you want to target mysql, you can:
- use EF core etl.net extensions by setting up your EF Core for mysql (https://dev.mysql.com/doc/connector-net/en/connector-net-entityframework-core.html). Read carefully the `important` note here: https://paillave.github.io/Etl.Net/docs/recipes/ef#save (about the use of `SaveMode.EntityFrameworkCore`)
- make your own version of extension for etl.net using the extension for SqlServer native: https://github.com/paillave/Etl.Net/tree/master/src/Paillave.Etl.SqlServer (if you know mysql, it should be fairly easy)

**Comment by paillave** [comment]
> I'm bumping into some trouble understanding how to debug it.

(sorry, for some reason, I missed this part of your message)
Can you tell me what problems you are facing?
